### ISSUE TITLE
how do I use vmui

### ISSUE THREAD

**Issue by clhedrick** [VictoriaMetrics/VictoriaMetrics#6679]
### Is your question request related to a specific component?

vmui

### Describe the question in detail

Is there any documentation for vmui? I'm using Victoria Metrics for sytem data, fed by telegraf using influx protocol. I can't get anything useful from vmui. Any query I type returns syntax error JSON status. The exporer wants app and instance which doesn't make any sense.

The documenntatnol doesn't say there's any mandatory configuration, but it certainy sounds like there must be.

### Troubleshooting docs

- [X] General - https://docs.victoriametrics.com/troubleshooting/
- [ ] vmagent - https://docs.victoriametrics.com/vmagent/#troubleshooting
- [ ] vmalert - https://docs.victoriametrics.com/vmalert/#troubleshooting

**Comment by xiaozongyang** [comment]
@clhedrick I foud this doc about vmui, is this helpful? https://docs.victoriametrics.com/single-server-victoriametrics/#vmui

**Comment by jiekun** [comment]
Hi. @clhedrick 

As @xiaozongyang mentioned you could find vmui docs from the link above.

> Any query I type returns syntax error JSON status. The exporer wants app and instance which doesn't make any sense.

Please provide more detail steps, screenshot of the issue you encountered, or some query statement you are using. I am not sure what kind of error you are facing, and whether it's related to data ingestion or querying.

Feel free to add more comments so that I can delve deeper into the issue for you.

**Comment by clhedrick** [comment]
I'm using the official container. It runs

/victoria-metrics-prod -retentionPeriod=90d

No query is needed. "Top queries" shows the same thing: Just navigating to the page gives me

VictoriaMetrics tracks the last search.queryStats.lastQueriesCount queries with durations at least search.queryStats.minQueryDuration

SyntaxError: Unexpected token 'r', "remoteAddr"... is not valid JSON

--version gives

victoria-metrics-20240424-101603-heads-fork-allow-build-on-arm64-0-g01d9c83ceb

(I note that this is an Intel Gold 6148, not an arm. Ubuntu 24.04)

**Comment by clhedrick** [comment]
If it matters, I'm going through vmauth.

- username: xxx
  password: xxx
  url_map:
    - src_paths:
       - "/vmui"
       - "/vmui/.*"
      url_prefix: "http://victoriametrics:8428"

**Comment by clhedrick** [comment]
Yes, it does matter. When I don't go through vmauth, it seems to work, though I still don't know what job and instance to use for the explorer.

**Comment by jiekun** [comment]
I'm removing `vmui` and `docs` label. As those issues are not relevant. 

> though I still don't know what job and instance to use for the explorer.

I'm not sure what you are referring to. Please provide a more detailed and complete description of the issue you are facing.

**Comment by clhedrick** [comment]
It appears to be a bug (or oundocumented feature) in vmauth.

As to job and instance. I'm trying to explore metrics submitted from telegraf. I have no idea what job or instance means in that context.

**Comment by nonproto** [comment]
I was seeing a similar and it seems the UI has a bug allowing routing when missing /select.

`vmselect.mydomain/0/vmui` brings up a basic html page where you have a list of end points.  click the vmui brings you to
`vmselect.mydomain/0/vmui/vmui` this looks like the correct ui but upon clicking explore and any tab it show `SyntaxError: Unexpected token 'r', "remoteAddr"... is not valid JSON`

however looking at the docs i should have been going to 
`vmselect.mydomain/select/0/vmui` this works perfectly fine

**Comment by jiekun** [comment]
Hello @nonproto & @clhedrick are you still observing similar issue?

**Comment by nonproto** [comment]
I am version pinned currently with this issue
https://github.com/VictoriaMetrics/operator/issues/1094#issuecomment-2660459692

**Comment by jiekun** [comment]
@nonproto hmm, that's a different one. Could I close the [current one](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/667)?

**Comment by nonproto** [comment]
Was the UI pathing fixed on newer versions?

**Comment by jiekun** [comment]
I was not able to reproduce it. It would be great if you could test it in newer version. And hopefully if there's step to reproduce it please comment here!

**Comment by nonproto** [comment]
Seems to be resolved
`vmselect.mydomain` now shows just the docs url.
